apis/core/lib/domain/identifiers.ts

```diff
@@ -55,3 +55,7 @@ export function user(user: string): NamedNode {
 export function job(jobCollection: GraphPointer<NamedNode>): NamedNode {
   return $rdf.namedNode(`${jobCollection.value}/${nanoid()}`)
 }
+
+export function dimensionMetadata(dimensionMetadataCollection: DimensionMetadataCollection, mapping: ColumnMapping): NamedNode {
+  return $rdf.namedNode(`${dimensionMetadataCollection.pointer}?dimension=${mapping.pointer}`)
```

Comment:
We discussed with @martinmaillard and query params are not such a great choice after all.

Check the updated seed graph in [#200](https://github.com/zazuko/cube-creator/pull/200/files#diff-ee0d6661710ff05549447fce5560bf3fde360e65f8ea766d8a1ab77c1e40ebe9R50-R54)

I propose you simply concatenate the collection ID with the `targetProperty`. Keep the string value unchanged and maybe use `import { shrink } from '@zazuko/rdf-vocabularies'` to shorten URI target properties